content/post/cloud-init-cloud-on-hypriot-x64.md

```diff
@@ -0,0 +1,348 @@
++++
+Categories = ["Docker", "Raspberry Pi", "ARM", "ARM64"]
+Tags = ["Docker", "Raspberry Pi", "ARM", "ARM64", "External" "cloud-init", "rpi-64"]
```

Comment:
Here is a comma missing, which causes a build error!
Just change `"External"` into `"External",` and it will works